tests/bluetooth/audio/bap_base/Kconfig

```diff
@@ -0,0 +1,14 @@
+# Temporary override
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config BT_BAP_BASE
+	# Override until someone fixes the UT build system
+	bool
+	default y
+
+# Include Zephyr's Kconfig.
+source "Kconfig.zephyr"
```

Comment:
Welp :(

Will have a look at whatever caused these things once this is merged

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/77406
https://github.com/zephyrproject-rtos/zephyr/pull/77405